cryptofeed/rest/gdax.py

```diff
@@ -234,9 +259,90 @@ def get_orders(self, body):
         return self._get_orders(body)
 
 
-    def  get_order(self, order_id: str):
+    def get_order(self, order_id: str):
         return self._trade_normalization(self._get_order(order_id))
 
+    def get_accounts(self):
+        return self._get("/accounts")
+
+    def get_account(self, account_id: str):
+        endpoint = "/accounts/{}".format(account_id)
+        return self._get(endpoint)
+    
+    def get_account_history(self, account_id: str, pagination={}):
+        endpoint = "/accounts/{}/ledger".format(account_id)
+        endpoint = self._pagination(endpoint, pagination)
+        
+        return self._get(endpoint)
+    
+    def get_holds(self, account_id: str, pagination={}):
+        endpoint = "/accounts/{}/holds".format(account_id)
+        endpoint = self._pagination(endpoint, pagination)
+        
+        return self._get(endpoint)
+    
+    def deposit_funds(self, body):
+        """
+        data format
+        {
+            "amount": 10.00,
+            "currency": "USD",
+            "payment_method_id": "bc677162-d934-5f1a-968c-a496b1c1270b"
+        }
+        """
+        return self._post("/deposits/payment-method", body)
+
+    def deposit_coinbase(self, body):
```

Comment:
the other one is called deposit funds, why call this one "coinbase"? Why not `deposit_coins` ?

Reply:
because this is specifically letting you deposit from a different coinbase account, if they integrate with other crypto accounts then this will be how we distinguish...funds is just from any other payment method

Reply:
true, but then you'd end up with deposit_bitfinex, deposit_bitmex, deposit_coinbase, etc and they should all have standardized names. you already distinguish the exchange by the attr on the rest class, i dont think we need it on the method name too, do we?